plugins/core/CMakeLists.txt

```diff
@@ -69,7 +69,12 @@ target_link_libraries( viame_train_detector
                vital_exceptions
                vital_logger
                vital_algo
+               vital_util
                kwiversys
+               kwiver_algo_core
+               kwiver_algo_ocv
```

Comment:
see other note about depending on kwiver algos, you should just be linking against the vital base classes for these things

Reply:
Done.